specification/maps/data-plane/Render/stable/2024-04-01/render.json

```
@@ -0,0 +1,1022 @@
+                }
+              ]
+            }
+          }
+        ],
+        "produces": [
+          "image/jpeg",
+          "image/png"
+        ],
+        "responses": {
+          "200": {
+            "description": "This image is returned from a successful Get Map Static Image call.",
+            "schema": {
+              "type": "object",
+              "format": "file",
+              "readOnly": true
+            },
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "description": "The content-type for the response."
+              }
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/map/copyright/bounding/{format}": {
+      "get": {
+        "summary": "Use to get copyright information for the specified bounding box.",
+        "description": "\n\nReturns copyright information for a given bounding box. Bounding-box requests should specify the minimum and maximum longitude and latitude (EPSG-3857) coordinates",
+        "operationId": "Render_GetCopyrightFromBoundingBox",
+        "x-ms-examples": {
+          "Successful BoundingBox Copyright Request": {
+            "$ref": "./examples/Render_GetCopyrightFromBoundingBox.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/BoundingBoxSouthWest"
+          },
+          {
+            "$ref": "#/parameters/BoundingBoxNorthEast"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/IncludeText"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Copyright"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/map/copyright/tile/{format}": {
+      "get": {
+        "summary": "Use to get copyright information.",
+        "description": "\n\nReturns the copyright information for a given tile. To obtain the copyright information for a particular tile, the request should specify the tile's zoom level and x and y coordinates. For more information, see: [Zoom Levels and Tile Grid](/azure/azure-maps/zoom-levels-and-tile-grid).\n\nCopyrights API is designed to serve copyright information for Render service. In addition to basic copyright for the whole map, API is serving specific groups of copyrights for some countries/regions.",
```

The first sentence may not be required given the summary field basically says the same thing. Also, no ':' is needed when referring to another article for more information.

```suggestion
        "description": "\n\nTo obtain the copyright information for a particular tile, the request should specify the tile's zoom level and x and y coordinates. For more information, see [Zoom Levels and Tile Grid](/azure/azure-maps/zoom-levels-and-tile-grid).\n\nCopyrights API is designed to serve copyright information for Render service. In addition to basic copyright for the whole map, API is serving specific groups of copyrights for some countries/regions.",
```